src/sagas/index.js

```diff
@@ -107,7 +109,7 @@ export function* transformAndPermitVm (vm) {
 
   // Permit disks fetched and transformed along with the VM
   for (const disk of internalVm.disks) {
-    disk.userPermits = yield entityPermissionsToUserPermits(disk)
+    disk.userPermits = entityPermissionsToUserPermits(disk)
```

Comment:
Based on previous suggestion:

```suggestion
    disk.userPermits = _entityPermissionsToUserPermits(disk)
```